### PR TITLE
Adding geolocation in response. Class types Station and StationRespon…

### PR DESCRIPTION
**Pull request by antoniovlx**
Hi,

Congratulations for a fantastic tool for helping with the use of the API.

I think It would be great if _geo_lat_ and _geo_long_ fields will be available after the normalization of the stations.

Changes I've made:
Adding geolocation fields in normalization of response. Class types _Station_ and _StationResponse_ modified with optional fields _geo_lat_ and _geo_long_.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -99,7 +101,9 @@ export type Station = {
   lastLocalCheckTime: Date
   clickTimestamp: Date
   clickCount: number
-  clickTrend: number
+  clickTrend: number,
+  geo_lat?: number
+  geo_long?: number
```
Comment (ivandotv) [inline src/constants.ts:106]: New properties should use camel case like the rest,
```suggestion
  geoLat?: number
  geoLong?: number
```

**Review by ivandotv — CHANGES_REQUESTED** [review]
Than for taking the time to contribute.

All properties should use camel case naming convention.
Also, the tests are failing because of mock station and mock response mismatch.

File: src/radioBrowser.ts [hunk 2]
```diff
@@ -267,6 +267,8 @@ export class RadioBrowserApi {
         lastLocalCheckTime: new Date(response.lastlocalchecktime),
         language: response.language.split(','),
         lastCheckTime: new Date(response.lastchecktime),
+        geo_lat: response.geo_lat !== null ? response.geo_lat: undefined,
```
Comment (ivandotv) [inline src/radioBrowser.ts:270]: You forgot to change the properties to camel case.

Comment (antoniovlx) [reply src/radioBrowser.ts:270]: Sorry

**Comment by ivandotv** [comment]
Thanks! 👏

**Pull request merged**